examples/tensorflow2/scripts/tf_keras_gradienttape.py

```diff
@@ -0,0 +1,90 @@
+"""
+This script is a ResNet training script which uses Tensorflow's Keras interface.
+It has been orchestrated with SageMaker Debugger hook to allow saving tensors during training.
+Here, the hook has been created using its constructor to allow running this locally for your experimentation.
+When you want to run this script in SageMaker, it is recommended to create the hook from json file.
+Please see scripts in either 'sagemaker_byoc' or 'sagemaker_official_container' folder based on your use case.
```

Comment:
relative path to folder.

Reply:
Have to remove this comment. I structured the directory similar to mxnet and pytorch. This comment is coming from tensorflow directory